bin/bossSwitch.py

```diff
@@ -157,13 +179,30 @@ def create_help(header, options):
                                      epilog=actions_help + scenario_help)
     parser.add_argument("--aws-credentials", "-a",
                         metavar = "<file>",
-                        default = "../config/aws-credentials",
+                        default = "aws-credentials",
                         help = "File with credentials to use when connecting to AWS (default: AWS_CREDENTIALS)")
     parser.add_argument("--scenario",
```

Comment:
You can remove all references to scenarios now that you're just taking the path to the asg config file directly.

Reply:
I thought I'd leave them, since they might be of use in the future?

Reply:
Removed references though, to keep code clean